src/components/organisms/SideNav/index.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { contentList } from "../../../data/SideNavList";
+import SideNav from ".";
+
+test("Renders the SideNav", () => {
+  render(<SideNav contentList={contentList} />);
+  const ReactElement = screen.getByTestId("side-nav-bar");
+  expect(ReactElement).toBeInTheDocument();
+});
+
+test("Check templates tab", () => {
+  render(<SideNav contentList={contentList} />);
```

Comment:
You can also test for on click functionality for tabs

Reply:
We don't have any onclick functionality on tabs. It's static